src/Markup/Avalonia.Markup.Xaml.Loader/CompilerExtensions/Transformers/AvaloniaXamlIlWellKnownTypes.cs

```diff
@@ -52,7 +53,18 @@ class AvaloniaXamlIlWellKnownTypes
         public IXamlType Uri { get; }
         public IXamlType FontFamily { get; }
         public IXamlConstructor FontFamilyConstructorUriName { get; }
-        
+        public IXamlType Thickness { get; }
+        public IXamlConstructor ThicknessFullConstructorName { get; }
```

Comment:
The `Name` in `FontFamilyConstructorUriName` wasn't a suffix, it was referring to `FontFamily(Uri uri, string name)` signature.

Reply:
Right, I guess I've copy pasted without thinking much about name here :)